classifiers/python_functions/emotionality_detection/code_snippet.md

```diff
@@ -4,13 +4,13 @@ from LeXmo import LeXmo
 import nltk
 nltk.download('punkt')
 
-YOUR_ATTRIBUTE = "text-attribute"
+YOUR_ATTRIBUTE: str = "text"
 
 def emotionality_detection(record: Dict[str, Any]):
     text = record[YOUR_ATTRIBUTE].text # SpaCy document, hence we need to call .text to get the string
     emo = LeXmo.LeXmo(text)
     emo.pop("text", None)
     emo = max(emo, key=emo.get)
     
-    return {"emotions": emo}
+    return emo
```

Comment:
do we have lit of emotions this can return so a user could prepare a labeling task so this doesn't run into errors?

Reply:
`LeXmo` returns a dictionary of emotions with scores and in this module we return the emotion with the highest score. However, it can also be good to include a list of the available emotions: [anger, fear, anticipation, trust, surprise, sadness, joy, disgust]

Reply:
Added a comment to the source code as well as the README.md with the list of possible emotions.